Libraries/LibJS/Interpreter.h

```diff
@@ -59,9 +59,25 @@ struct ScopeFrame {
 };
 
 struct CallFrame {
+    explicit CallFrame(Heap& heap)
+        : function_name()
+        , this_value()
+        , arguments(heap)
+        , environment(nullptr)
+    {
+    }
+
+    CallFrame(FlyString&& function_name, Value&& this_value, MarkedValueList&& arguments, LexicalEnvironment* environment)
+        : function_name(move(function_name))
+        , this_value(move(this_value))
+        , arguments(move(arguments))
+        , environment(environment)
+    {
+    }
+
     FlyString function_name;
     Value this_value;
-    Vector<Value> arguments;
+    MarkedValueList arguments;
```

Comment:
This is also already marked by `Interpreter::gather_roots()`.

Reply:
So is everything in a `CallStack`.